packit_service/worker/helpers/testing_farm.py

```diff
@@ -1146,15 +1196,12 @@ def _parse_comment_pr_argument(self) -> Optional[Tuple[str, str, str]]:
         if len(namespace_repo) != 2:
             logger.debug(
                 "Unexpected format of the test argument: "
-                f"not able to split the test argument {pr_argument} with '/'."
+                f"not able to split the test argument "
+                f"{self.comment_arguments.pr_argument} with '/'."
             )
             return None
         namespace, repo = namespace_repo
 
-        logger.debug(
```

Comment:
why removing this? :thinking:

Reply:
Guess I removed it because I wanted to move it to a different place. I will put it back.